.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!--
```

Comment:
Again, this is awesome first step! Here is what we use at work which has helped immensely. Feel free to take/use ideas. Just sharing possibilities :)

**Description:** 
<!-- Insert descriptive description here about the pull request -->
* 

**Issue(s):** 
<!-- Link to the issue that this pull request resolves -->
* #ISSUE#

<!-- if applicable 
**Documentation:**
* [Documentation](https://www.example.com/foo/bar)
 -->